docker/k8s/vtadmin/nginx.conf

```diff
@@ -0,0 +1,11 @@
+events {}
+http {
+    include mime.types;
+    server {
+        listen 14201;
+        root /var/www;
```

Comment:
we should add headers + gzip for caching static files expect for the `config.js` file

Reply:
something like this (not tested)

```
gzip on;
gzip_disable "msie6";

gzip_vary on;
gzip_proxied any;
gzip_comp_level 6;
gzip_buffers 16 8k;
gzip_http_version 1.1;
gzip_min_length 256;
gzip_types
  application/atom+xml
  application/geo+json
  application/javascript
  application/x-javascript
  application/json
  application/ld+json
  application/manifest+json
  application/rdf+xml
  application/rss+xml
  application/xhtml+xml
  application/xml
  font/eot
  font/otf
  font/ttf
  image/svg+xml
  text/css
  text/javascript
  text/plain
  text/xml;

location = /config/config.js {
        expires off;
    }

    location ~* \.(css|gif|jpg|js|png|ico|otf|sng|xls|doc|exe|jpeg|tgx)$ {
        expires max;
    }

    location / {
       client_max_body_size 50M;
       try_files $uri /index.html;
    }
 ```

Reply:
The committed config works for caching: 
![image](https://user-images.githubusercontent.com/15112506/180294035-5be04919-4ffc-4a16-99a3-7a20360c4167.png)

Reply:
added the gzip types and those have the correct gzip response headers: 
<img width="1107" alt="image" src="https://user-images.githubusercontent.com/15112506/180295378-cac0657c-0d48-4f07-ae72-1f468b7da04c.png">

Reply:
it works well 👍